mlflow/protos/service.proto

```diff
@@ -978,9 +978,23 @@ message GetMetricHistory {
   // Name of the metric.
   optional string metric_key = 2 [(validate_required) = true];
 
+  // Token indicating the page of metric history to fetch
+  optional int64 page_token = 4 [default = 0];
```

Comment:
Page token should be a string. Generally, we use an encoded byte representation of a proto.